windows/win_hostname.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+DOCUMENTATION = '''
+---
+module: win_hostname
+version_added: "2.0"
+short_description: Changes the hostname of a windows machine
+description:
+	- Changes the Hostname of a windows machine
+options:
+  name:
+    description:
+	  - New Hostname of the windows machine 
+	  required: true
+	  default: null
+	  aliases: []
+  restart:
+    description:
+	  - Restart after name change
+	required: false
+	choices:
+	- yes
+	- no
+	default: null
+	aliases: []
+  domain:
+    description:
+      - When specified will join a computer to the domain, requires domain_user and domain_pass
```

Comment:
this should state the fqdn, L70 is actually just for authentication purposes. right now it seems I can only authenticate with NLTM